201-userdefined-routes-appliance/azuredeploy.json

```diff
@@ -0,0 +1,553 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
It'd be a good idea to format this file using a JSON formatter for better readability. Here's a link - https://www.bing.com/search?q=json+formatter

Just paste in the text in this file